BDArmory/GameModes/Waypoints/CourseBuilderGUI.cs

```diff
@@ -0,0 +1,679 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using BDArmory.Competition.OrchestrationStrategies;
+using BDArmory.GameModes.Waypoints;
+using BDArmory.Settings;
+using BDArmory.Utils;
+using BDArmory.VesselSpawning;
+using System.Linq.Expressions;
```

Comment:
`System.Linq.Expressions` doesn't appear to be being used.

Reply:
Thanks, Visual Studio.